src/main/java/wolox/training/models/User.java

```diff
@@ -36,6 +37,7 @@ public Long getId() {
   }
 
   public void setId(long id) {
+    Preconditions.checkNotNull(id);
```

Comment:
id will never be null because you're using long and not Long.
Also, you can assign the return of the checkNotNull. That doesn't apply to checkArgument though.